framework/validators/FileValidator.php

```diff
@@ -258,7 +259,12 @@ protected function validateValue($file)
      */
     public function getSizeLimit()
     {
+        // Max POST size minus some bytes for form data 
+        $post_limit = $this->sizeToBytes(ini_get('post_max_size')) - 1024;
```

Comment:
In general one should configure POST size to be bigger than upload file size in PHP config. maybe we should just do a check and log a warning instead of using this value for guessing the max size?